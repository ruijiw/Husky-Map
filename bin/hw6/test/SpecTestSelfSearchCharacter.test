# Find a path using the LOLDictionary dataSet.
# This test uses both the parser and BFS algorithm in the MarvelPaths
# class and applies to a targeted dataset. This test is for the
# user asks for the character to itself case.

LoadGraph dictionary LOLDictionary.tsv

FindPath dictionary TEEMO TEEMO

FindPath dictionary JINX JINX