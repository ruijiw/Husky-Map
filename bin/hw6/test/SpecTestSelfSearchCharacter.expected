# Find a path using the LOLDictionary dataSet.
# This test uses both the parser and BFS algorithm in the MarvelPaths
# class and applies to a targeted dataset. This test is for the
# user asks for the character to itself case.

loaded graph dictionary

path from TEEMO to TEEMO:

path from JINX to JINX: